diff --git a/src/4cade.a b/src/4cade.a
index 36397bac9..d65521fff 100644
--- a/src/4cade.a
+++ b/src/4cade.a
@@ -151,6 +151,9 @@ ResetVector                          ; 6 bytes, copied to $100
 gPreloadStatus
          !byte %00000000
 
+gBootToAttractOnce
+         !byte %00000001
+
 gMegaAttractModeFilter    ; module types to include in mega-attract mode
          !byte %11111111
 ;               |||||||+- bit 0 include self-running demos
diff --git a/src/ui.search.mode.a b/src/ui.search.mode.a
index 9f1fd552c..9cce9a06a 100644
--- a/src/ui.search.mode.a
+++ b/src/ui.search.mode.a
@@ -118,6 +118,12 @@ InputKeyDispatch
 ; out:   never returns to caller (may JMP to other major modes)
 ;------------------------------------------------------------------------------
 SearchMode
+         lda   gBootToAttractOnce
+         beq   +                     ; only divert on first boot
+         lda   #$00
+         sta   gBootToAttractOnce
+         jmp   MegaAttractMode
++
          ldx   #$FF
          txs
          stx   gGameToLaunch         ; $FFFF = no game selected
